spirv_msl.cpp

```diff
@@ -5643,6 +6207,228 @@ void CompilerMSL::emit_instruction(const Instruction &instruction)
 	previous_instruction_opcode = opcode;
 }
 
+// If the underlying resource has been used for comparison then duplicate loads of that resource must be too
+static inline bool image_opcode_is_sample_no_dref(Op op)
```

Comment:
This function probably does not exist anymore.

Reply:
It was used in `CompilerMSL::emit_texture_op` but after I simplified this function to use `to_texture_op` I reverted it.